File: tests/src/test/java/com/optivem/atddaccelerator/templategenerator/systemtest/util/Language.java
package com.optivem.atddaccelerator.templategenerator.systemtest.util;

public enum Language {
    NONE("none"),
    JAVA("java"),
    DOTNET("dotnet"),
    TYPESCRIPT("typescript");

    private final String value;

    Language(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }

    public static Language[] getAll() {
        return new Language[] { JAVA, DOTNET, TYPESCRIPT };
    }
}

File: console/tests/Util/Language.cs
namespace Optivem.AtddAccelerator.TemplateGenerator.SystemTests.Util
{

    public static class Language
    {
        public const string None = "none";
        public const string Java = "java";
        public const string DotNet = "dotnet";
        public const string TypeScript = "typescript";
    }

    public static class LanguageExtensions
    {
        public static string[] GetAll()
        {
            return new[] { Language.Java, Language.DotNet, Language.TypeScript };
        }
    }
}